lisa/features/gpu.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from enum import Enum
+from typing import Any, cast
+
+from lisa.feature import Feature
+from lisa.operating_system import Linux, Redhat, Ubuntu
+from lisa.tools import Uname, Wget
+from lisa.util import LisaException
+
+FEATURE_NAME_GPU = "Gpu"
+
+
+class compute_sdk(Enum):
```

Comment:
Refer to other place in the codebase to create an enum.

Reply:
> Refer to other place in the codebase to create an enum.

Done